ocp4/secrets/secrets.go

```diff
@@ -45,6 +51,21 @@ func GenSecretFile(name string, encodedSecret string, namespace string) *Secret
 	return &secret
 }
 
+func buildData(idenityProviderType, encodedSecret string) interface{} {
```

Comment:
I believe this way of building secret data could be reusable for all oauth providers